source/use-cases/sensitive-data-encryption.txt

```diff
@@ -75,20 +75,106 @@ velit congue vel.
+                Document patient = new Document();
+                patient.append("name", name);
+                patient.append("ssn", ssn);
+                patient.append("bloodType", bloodType);
+                patient.append("medicalRecords", medicalRecords);
+                Document insurance = new Document();
+                insurance.append("policyNumber", policyNumber);
+                insurance.append("provider", provider);
+                patient.append("insurance", insurance);
+                collection.insertOne(patient);
+            }
+
+
+When a new record is inserted using a client configured to use CSFLE,
+such as when a doctor inserts a new patient in the MedcoMD Management
+System, the sensitive fields become encrypted:
+
+.. code-block:: json 
+
+    {
+        "_id": "5d7a7bbe6d58fd263b6d7315",
+        "name": "Jon Doe",
+        "ssn": "Ac+ZbPM+sk7gl7CJCcIzlRAQUJ+uo/0WhqX+KbTNdhqCszHucqXNiwqEUjkGlh7gK8pm2JhIs/P3//nkVP0dWu8pSs6TJnpfUwRjPfnI0TURzQ==",
+        "bloodType": "As+ZbPM+sk7gl7CJCcIzlRACESwHCTCtK/lQV9kF6/LRoL3mh59gzBVA42vGBVfLIycYWpfAy7ZCi2eRGEgMX5CrGl259Wfu6Zf/ELBVqQDnyQ==",
+        "medicalRecords": "As+ZbPM+sk7gl7CJCcIzlRAEFt249toVYOlvlC/79cAtQ5jvE/ukF1ZLxRZn1g0zBBtPnf6L0AFTKMVdNJnjMGPMTszYU58qRE9uMvCU05DVHYl8DJnbtGXXFRLJ7ElQOc=",
+        "insurance": {
+            "provider": "MaestCare",
+            "policyNumber": "Ac+ZbPM+sk7gl7CJCcIzlRAQm7kFhN1hy3l7Wt3BSpBMbvVSuiaDsf3UPF9bvJLTEcC+Ka+3kZI4SVZinj4tyc5uDYeyh6+7phpKrQo4CHWyg=="
+        }
+    }
+
+When a new record is inserted using a client that is not configured to
+use CSFLE, such as when a receptionist inserts a new patient, all fields
+are stored as plain-text:
+
+
+.. code-block:: json 
+
+    {
+        "_id": "5d696fff112d8c439bae7068",
+        "name": "Jane Doe",
+        "ssn": 1112223333,
+        "bloodType": "a-",
+        "medicalRecords": [
+            {
+                "weight": 120,
+                "bloodPressure": "120/80"
+            }
+        ],
+        "insurance": {
+            "provider": "PlatinumCare North East",
+            "policyNumber": 4410
+        }
+    }
+
+.. note::
+    All the data the receptionist inserts is unencrypted even if it is a
+    sensitive field, such as SSN. You may want to include application
+    logic preventing clients without CSFLE configured from writing to
+    sensitive fields. An alternative approach is to use Server Side
+    Field Level Encryption in combination with CSFLE. Doing so adds
+    further security by throwing an error if an attempt is made to
+    insert plaintext into a sensitive field. 
+
+You can read the document(s) using traditional MongoDB driver methods as
+well. In the following example, a doctor in the MedcoMD management
+system scans a patient’s wristband. A function is called to search via
+the SSN of the scanned wristband, and the following results are returned:
```

Comment:
Could we reword this to use active voice (instead of "a function is called...")? I think you could combine a phrase with the previous sentence to link the action ("scan a wristband") and the results ("queries MongoDB which returns..."). 
 
Also, did you mean "SSN on the wristband"? If you use "of", it implies an SSN belongs to a wristband rather than a patient.

Reply:
*  I'm not entirely sure how to reword this sentence while retaining clarity and ease of understanding. Can you give any suggestions? 
* Fixed, the sentence now reads: "A function is called to search via the SSN on the scanned wristband ..."

Reply:
Resolving this since this is outdated & that sentence is no longer in the guide